.gitignore

```diff
@@ -36,8 +36,12 @@ nbproject
 .directory
 plugvenv/
 venv/
+
 # Pytest files
 tests/rsc/camera/debug/[^R]*
 tests/rsc/camera/out.jpg
 tests/logs/*.txt
 *pytest-logs.txt
+
+# node modules
+node_modules
```

Comment:
why we need node_modules?

Reply:
This will appear once we add javascript tests and we shouldn't upload this to the repo

Reply:
But this is not related to the docker image and you are already merging into alpha